src/components/VAudioTrack/VWaveform.vue

```diff
@@ -149,6 +149,14 @@
     >
       {{ message }}
     </div>
+
+    <!-- Seek disabled message overlay -->
+    <div
+      v-if="seekDisabledNotice"
+      class="invisible group-hover:visible group-focus:visible absolute w-full inset-0 flex items-center justify-center font-bold text-xsm bg-yellow/75 z-40"
+    >
+      {{ seekDisabledNotice }}
+    </div>
```

Comment:
Imo we use the existing `message` field, with a prop that allows it to be transient. I doubt the two will ever need to overlap.

Reply:
At a minimum they could definitely share some styles; good suggestion.